docs/dashboard/access-roles.md

```diff
@@ -0,0 +1,53 @@
+<!--
+title: Serverless Dashboard - Access Roles
+menuText: Access Roles
+layout: Doc
+-->
+
+<!-- DOCS-SITE-LINK:START automatically generated  -->
+
+### [Read this on the main serverless docs site](https://www.serverless.com/framework/docs/)
```

Comment:
Docs site link needs to be updated to match the link for this page in docs/README.md

Reply:
What's the issue here @ganeshvlrk ? These are unchanged, I think.